src/components/css/Book.module.css

```diff
@@ -0,0 +1,64 @@
+.book-container {
+  display: flex;
+  margin: 0 10px;
+  margin-bottom: 40px;
+}
```

Comment:
- [ ] Great job so far @Rishi-Mishra0704 I see you tried to implement the previous reviewer's suggestion but some of them weren't done properly and in the process, some elements lost their original styles but no need to worry I have made sure to point out each of them in the screenshot below :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/232098771-09a2fbe7-cc11-4c3f-9c02-f711e2ac1b3b.png)

- The Original Design
![image](https://user-images.githubusercontent.com/102699958/232098984-f8610b61-5b6f-4d3f-a685-aa1deb91093b.png)